index.js

```diff
@@ -173,22 +188,28 @@ DB.prototype.del = function (key, opts, cb) {
   })
 }
 
-DB.prototype._del = function (key, opts, cb) {
+// OsmVersion, Opts -> [OsmBatchOp]
+DB.prototype._getDocumentDeletionBatchOps = function (key, opts, cb) {
```

Comment:
See my proposed patch: https://gist.github.com/gmaclennan/112c6786511c63c406f2eb64d4157b9d

This does two things:

1. I don't think there is any need to map over the docs
2. Add `v.type` to deletes